src/game_inventory.cpp

```diff
@@ -487,7 +487,24 @@ item_location game_menus::inv::disassemble( Character &p )
                          _( "You don't have any items you could disassemble." ) );
 }
 
+//for testing purposes
```

Comment:
Rather than adding a whole separate function just for the tests, it would be better to adjust the test to cover the real in-game function. The test cases already have a `Character &u` reference; why not simply pass that to the invocations of `satiety_bar` from `tests/comestible_test.cpp`?

Reply:
For example, you can change the test:

```cpp
CHECK( satiety_bar( 396 ) == "<color_c_yellow>||</color>..." );
```

to this:

```cpp
CHECK( satiety_bar( 396, u ) == "<color_c_yellow>||</color>..." );
```

Do the same for all the others, since you have changed the signature of `satiety_bar`. Then you can get rid of the duplicate function definition.

Reply:
I shouldn't be coding at 1 am. Fixed.